### ISSUE TITLE
Telegraf fails to connect to OpenTSDB in HTTP Mode

### ISSUE THREAD

**Issue by skrud** [influxdata/telegraf#1769]
## Bug report

I'm been unable to connect to OpenTSDB in HTTP Mode using Telegraf 1.0.0. I've tried several different steps that I've outlined below.

### 1. Setting "useHttp" in config
According to the [README](https://github.com/influxdata/telegraf/tree/master/plugins/outputs/opentsdb), I'm supposed to set "useHttp = true" in the config. I've tried this using camel case and also snake case with the same results:

    Error parsing telegraf.conf, line 83: field corresponding to `useHttp' is not defined in `*opentsdb.OpenTSDB'
    Error parsing telegraf.conf, line 83: field corresponding to `use_http' is not defined in `*opentsdb.OpenTSDB'

### 2. Using host = "http://..."
In the [source code](https://github.com/influxdata/telegraf/blob/master/plugins/outputs/opentsdb/opentsdb.go#L33-L35), it suggests prefixing the host name with `http://` in order to use http mode. When I do this, I get an error that says:

    2016/09/14 19:29:07 Failed to connect to output opentsdb, retrying in 15s, error was 'OpenTSDB: TCP address cannot be resolved'
    2016/09/14 19:29:22 OpenTSDB: TCP address cannot be resolved

I have tried this with IP Addresses (127.0.0.1, and 192.168.33.10) as well as simply "http://localhost".

### Relevant telegraf.conf:

```
# Configuration for OpenTSDB server to send metrics to
[[outputs.opentsdb]]
  ## prefix for metrics keys
  #prefix = "my.specific.prefix."

  ## Telnet Mode ##
  ## DNS name of the OpenTSDB server in telnet mode
  host = "http://localhost"

  ## Port of the OpenTSDB server in telnet mode
  port = 4242

  ## Debug true - Prints OpenTSDB communication
  debug = true
```

### System info:
Telegraf v1.0.0
OpenTSDB v2.2.0

I've run on two different OSes:
* Mac OS X El Capitan, connecting to an OpenTSDB instance running in a Vagrant VM. (Telegraf is v1.0.0 but installed from Homebrew)
* On a Vagrant VM running Ubuntu, the same host that's running OpenTSDB locally. (Installed the .deb file hosted on the official download page).

### Steps to reproduce:

1. Create a Telegraf.conf with an opentsdb section
2. Configure opentsdb to use http (by putting http:// prefix in the host field, or providing the use_http = true config field).
3. Start telegraf

### Expected behavior:

Telegraf should connect to OpenTSDB and publish metrics to it.

### Actual behavior:

Telegraf fails to connect to OpenTSDB and shuts down after the retry attempt failed.

**Comment by sparrc** [comment]
sorry but that feature isn't available in 1.0, it'll be available in 1.1 and in the meantime you'll need to use a nightly build, see https://github.com/influxdata/telegraf/blob/master/CHANGELOG.md

**Comment by skrud** [comment]
My bad, I hadn't noticed it wasn't in the release. I'll check out a nightly in the meantime. Thanks!
